cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaBrokerConfigurationBuilder.java

```diff
@@ -134,6 +135,25 @@ public KafkaBrokerConfigurationBuilder withCruiseControl(String clusterName, Cru
         return this;
     }
 
+    /**
+     * Configures the Strimzi Metrics Reporter. It is set only if user enabled Strimzi Metrics Reporter.
+     *
+     * @param model     Strimzi Metrics Reporter configuration
+     *
+     * @return Returns the builder instance
+     */
+    public KafkaBrokerConfigurationBuilder withStrimziMetricsReporter(StrimziMetricsReporterModel model)   {
+        if (model != null && model.isEnabled()) {
+            printSectionHeader("Strimzi Metrics Reporter configuration");
+            writer.println(StrimziMetricsReporterConfig.LISTENER_ENABLE + "=true");
+            writer.println(StrimziMetricsReporterConfig.LISTENER + "=http://0.0.0.0:" + StrimziMetricsReporterModel.METRICS_PORT);
```

Comment:
Doesn't this mean that the reporter would work only for IPv4 envs/dual where IPv4 is preferable? Shouldn't we also add listener:
```java
writer.println(StrimziMetricsReporterConfig.LISTENER + "=http://[::]:" + StrimziMetricsReporterModel.METRICS_PORT);
```
?
or something like that?

Reply:
@see-quick thanks for bringing this up.

AFASIK, unless `java.net.preferIPv4Stack=true` is set, the Java runtime always binds to IPv6 anylocal address (::) on a dual-stack machine, which can accept both IPv6 and IPv4 traffic. This means that `0.0.0.0` should be fine and more readable. Anyway, it would be good to test this PR on a IPV6-only machine if you can.

Reply:
you can use `/packit test --labels ipv6` to run tests on Kind configured with ipv6 only

Reply:
Thanks. The problem is that we only have unit tests at this stage.

Reply:
Well, we should have at least some IT's before merging this, I think.

Reply:
This behavior is tested in the metrics-reporter project. Next release will fix the issue that requires to specify the anylocal address, so we should be good.